src/script_menu.c

```diff
@@ -283,7 +281,7 @@ bool8 ScriptMenu_YesNo(u8 left, u8 top)
     {
         gSpecialVar_Result = 0xFF;
         DisplayYesNoMenuDefaultYes();
-        taskId = CreateTask(Task_HandleYesNoInput, 0x50);
+        CreateTask(Task_HandleYesNoInput, 0x50);
```

Comment:
```suggestion
        (void) CreateTask(Task_HandleYesNoInput, 0x50);
```

Reply:
I don't really think this is needed here.

Reply:
I was pretty sure it would warn on `pedantic` but maybe I was thinking about c++ 🤔